user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml

```diff
@@ -90,6 +93,38 @@ local:
       - 'Estimated GPU Duration'
       - 'App Duration'
       - 'Estimated GPU Savings(%)'
+    processDFProps:
+      minimumWorkerCount: 2
+      gpuScaleFactor: 0.80
```

Comment:
we may want this to be in each CSP config as we would ideally tune per CSP based on benchmarks